modular_skyrat/master_files/code/modules/uplink/uplink_items.dm

```diff
@@ -362,13 +362,13 @@
 	cost = 20
 
 //Badass section down here
-/datum/uplink_item/loadout_skyrat/robohand
-	name = "Robohand Bundle"
-	desc = "Themed after the infamous terrorist(or not), Johnny Robohand. You have no reason to fail your objectives with this kit. The gun inside requires your arm to be robotic. \
-			It comes with a robotic replacement arm. Wake the fuck up, samurai."
-	item = /obj/item/storage/backpack/duffelbag/syndie/loadout/robohand
-	cost = 35
-
-/datum/uplink_item/loadout_skyrat/robohand/purchase(mob/user, datum/component/uplink/U)
-	. = ..()
-	notify_ghosts(message = "[user] has purchased the Johnny Robohand bundle, watch him be a badass!", ghost_sound = 'modular_skyrat/modules/3516/sound/wakeup.ogg', source = user) //Everyone needs to know he's a badass
+///datum/uplink_item/loadout_skyrat/robohand
+//	name = "Robohand Bundle"
+//	desc = "Themed after the infamous terrorist(or not), Johnny Robohand. You have no reason to fail your objectives with this kit. The gun inside requires your arm to be robotic. \
+//			It comes with a robotic replacement arm. Wake the fuck up, samurai."
```

Comment:
```suggestion
//	desc = "Themed after the infamous terrorist(or not), Johnny Robohand. You have no reason to fail your objectives with this kit. The gun inside requires your arm to be robotic. It comes with a robotic replacement arm. Wake the fuck up, samurai."
```

Reply:
Not sure why this description was originally two-line but it shouldn't.
Yes I know you are commenting it out, should still fix it anyway